feature/interface/loopback/ate_tests/loopback_aggregate_test/README.md

```diff
@@ -0,0 +1,65 @@
+# RT-5.6: Interface Loopback mode
+
+## Summary
+
+Ensure Interface mode can be set to loopback mode and can be added as part of static LAG.
+
+## Procedure
+
+### TestCase-1:
+
+*   Configure DUT port-1
```

Comment:
At this point, We should ensure that we use port that is not connected to remote system, and therefore interface is in DOWN STATE.

Reply:
Updated test steps